.github/workflows/main.yml

```diff
@@ -55,6 +55,13 @@ jobs:
         with:
           name: python-artifacts
           path: .
+          
+      - name: Upload helm artifacts
+        if: github.ref_name == 'main'
```

Comment:
The condition `if: github.ref_name == 'main'` restricts the upload of helm artifacts to only when the 'main' branch is used. If you intend to upload helm artifacts for other branches as well, consider removing or modifying this condition.